### ISSUE TITLE
free/available/total space calculation

### ISSUE THREAD

**Issue by ip1981** [whamcloud/integrated-manager-for-lustre#1808]
Here in GUI https://github.com/whamcloud/integrated-manager-for-lustre/blob/31cf641cf649ef3dd2bd2bd75f98ee4ddb994708/iml-gui/crate/src/page/filesystem.rs#L434-L446
and in this PR https://github.com/whamcloud/integrated-manager-for-lustre/pull/1803

we calculate used space as the difference between total and free, it looks like the used space is always non-zero (hundreds of kilobytes in VBox) on a new filesystem (without files). I also suspect that eventually the used space will be greater than the available space which we show on charts and in command line.

So calculation is like this:

`used = total - free` and we show `used / available`

Should we instead do this:

`used = available - free`? (and still show `used / available`).

In other words, was this change incomplete https://github.com/whamcloud/integrated-manager-for-lustre/commit/756e3b3400c86102513e68ec050a4a62ab890ac6?

On the other side, existing calculation is in line with `df -h`, for example:
```
root@c1 ~]# df -h /mnt/*
Filesystem                            Size  Used Avail Use% Mounted on
10.73.20.11@tcp:10.73.20.12@tcp:/fs    47G  333M   44G   1% /mnt/fs
10.73.20.11@tcp:10.73.20.12@tcp:/fs2   47G  332M   44G   1% /mnt/fs2

```

**Comment by ip1981** [comment]
I am totally wrong :)
